### PR TITLE
add test for zoom client

### PR DESCRIPTION
**Pull request by ibez92**

### CODE REVIEW

File: internal/zoom/client.go [hunk 1]
```diff
@@ -53,6 +53,9 @@ func (c *Client) GetMeetingParticipants(meetingID string) ([]Participant, error)
 	if err != nil {
 		return nil, err
 	}
+	if pResp.Code >= 300 {
```
Comment (MOZGIII) [inline internal/zoom/client.go]:
```suggestion
	if pResp.Code != 200 {
```

Or whatever the response is supposed to be there.

File: internal/zoom/client_test.go [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+					}
+				]
+			}`)
+		}
+		w.Write(message)
+	})
+}
+
+func TestGetMeetingParticipants(t *testing.T) {
+	ts := httptest.NewServer(dummyHandler())
```
Comment (MOZGIII) [inline internal/zoom/client_test.go]: The way this is usually used is at each test case a server is started independently, and the test case for a particular test scenario is just written inline in the callback form handler.

```go
http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
	fmt.Fprintln(w, "Hello, client")
})
```

File: internal/zoom/cover.out [hunk 3]
```diff
@@ -0,0 +1,18 @@
+mode: set
```
Comment (MOZGIII) [inline internal/zoom/cover.out]: Test coverage artifacts shouldn't be in the repo

File: internal/zoom/client_test.go [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+	token := "token"
+	secret := "secret"
+	meetingID := "meetingID"
+	client := NewClient(token, secret, meetingID, ts.URL)
+
+	testCases := []TestCase{
+		{
+			MeetingID: "error",
```
Comment (MOZGIII) [inline internal/zoom/client_test.go]: It is better to test with real data to ensure handlers work properly. It is also important to check other expected attributes in the handler - like that the expected input is properly passed (request URL, headers, etc). Using an inline callback or factory of such callbacks helps.

File: internal/zoom/client.go (hunk 1 above)
Comment (ibez92) [reply internal/zoom/client.go]: fixed

File: internal/zoom/client_test.go (hunk 2 above)
Comment (ibez92) [reply internal/zoom/client_test.go]: fixed

File: internal/zoom/cover.out (hunk 3 above)
Comment (ibez92) [reply internal/zoom/cover.out]: fixed

File: internal/zoom/client_test.go (hunk 4 above)
Comment (ibez92) [reply internal/zoom/client_test.go]: Can you give me some example please?

**Pull request closed without merging**